Tools/src/create_rpm.sh

```diff
@@ -47,6 +47,12 @@ chmod ug+rx ${BUILD_ROOT}/SOURCES/opt/aws/amazon-cloudwatch-agent/bin/amazon-clo
 chmod ug+rx ${BUILD_ROOT}/SOURCES/opt/aws/amazon-cloudwatch-agent/bin/start-amazon-cloudwatch-agent
 tar -zcvf ${BUILD_ROOT}/SOURCES/amazon-cloudwatch-agent.tar.gz -C ${BUILD_ROOT}/SOURCES opt etc
 
+echo "*************************************************"
+echo "Saving the tar file for Amazon Linux and RHEL ${ARCH}"
+echo "*************************************************"
+cp ${BUILD_ROOT}/SOURCES/amazon-cloudwatch-agent.tar.gz ${BUILD_SPACE}/bin/linux/${ARCH}/amazon-cloudwatch-agent.tar.gz
```

Comment:
It would be better to provide a constant since we use in multiple different places. Therefore, avoid human error and easier in reading.